Automattic/sensei
# Fix the email template editor style

Fix #6716 

## Proposed Changes
* Load the email editor style on the site editor
* Add a new CSS file to be loaded only on the site editor


## Testing Instructions
**Test 1:**
1. Go to the Branch
2. Run `npm run start`
3. Go to the `wp-admin/site-editor.php` 
4. Select the Sensei Email Template
5. Check if the rendered template looks like the attached image

**Test 2:**
6. Go to our new settings page (n/admin.php?page=sensei-settings&tab=email-notification-settings)
7. Select an email template. 
8. Select "Sensei Email"
9. Check if the site-editor style doesn't affected the templates.  


### Screenshots

**Before the change**
<img width="1331" alt="Screenshot 2023-03-28 at 13 30 46 (1)" src="https://user-images.githubusercontent.com/38718/228247224-8c7caa7c-4ddc-4996-8673-4c2b3650b2eb.png">


**After the change**
<img width="1615" alt="Screenshot 2023-03-28 at 14 02 00" src="https://user-images.githubusercontent.com/38718/228244738-23213b5a-f218-4201-950b-88113ec5c725.png">



## Pre-Merge Checklist
